common.blocks/demo/demo.js

```diff
@@ -65,37 +64,43 @@ modules.define('demo', [ 'i-bem__dom', 'pretty', 'functions__debounce' ], functi
             }
             this._result.setValue(finalCode);
         },
-        _save: function() {
+        save: function() {
             var template = this._getTemplate(),
-                bemjson = this._getBEMJSON();
+                bemjson = this._getBEMJSON(),
+                version = this._versionSelect.getValue(),
+                engine = this._engineSelect.getValue();
 
             store.set('playground', {
-                version: this.params.version,
+                version: version,
                 template: template,
-                bemjson: bemjson
+                bemjson: bemjson,
+                engine: engine
             });
 
             history.pushState({}, document.title, location.pathname + '?' + [
                 'template=' + encodeURIComponent(template),
-                'bemjson=' + encodeURIComponent(bemjson)
+                'bemjson=' + encodeURIComponent(bemjson),
+                'version=' + encodeURIComponent(version),
+                'engine=' + encodeURIComponent(engine)
```

Comment:
Can we set `template=…` and `bemjson=…` as last parameters? If the URL would be so long we don’t saw version and engine. And more over if google URL shortcut strips URL we didn't catch that.